core/services/llo/bm/dummy_transmitter.go

```diff
@@ -39,7 +38,7 @@ type transmitter struct {
 
 func NewTransmitter(lggr logger.Logger, fromAccount string) Transmitter {
 	return &transmitter{
-		lggr.Named("DummyTransmitter"),
+		logger.Named(lggr, "DummyTransmitter"),
```

Comment:
why has this cascaded to so many places? i only see a handful of NewEngine calls in this PR. does this PR intermix disparate concerns - are all these logger chains strictly necessary in this specific change?

Reply:
I followed the compiler errors, so all are necessary, unless we have cases where unrelated things are in the same file I suppose - but that seems like a separate issue.